src/main/java/org/opensearch/flowframework/constant/CommonValue.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.flowframework.constant;
+
+/**
+ * Representation of common values that are used across project
+ */
+public class CommonValue {
+
+    public static Integer NO_SCHEMA_VERSION = 0;
+    public static final String META = "_meta";
+    public static final String SCHEMA_VERSION_FIELD = "schema_version";
+    public static final Integer GLOBAL_CONTEXT_INDEX_SCHEMA_VERSION = 1;
+    public static final String GLOBAL_CONTEXT_INDEX_MAPPING = "{\n  "
+        + "   \"dynamic\": false,\n"
+        + "   \"_meta\": {\n"
+        + "       \"schema_version\": "
+        + GLOBAL_CONTEXT_INDEX_SCHEMA_VERSION
+        + "\n"
+        + "   },\n"
+        + "   \"properties\": {\n"
+        + "       \"pipeline_id\": {\n"
```

Comment:
let's change this to `workflow_id` to avoid confusion with ingest/search pipelines